ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/PageType.kt

```diff
@@ -0,0 +1,61 @@
+package com.braintreepayments.api.shopperinsights
+
+/**
+ * The page or view that a button is displayed on.
+ */
+enum class PageType {
+    /**
+     * The homepage
+     */
+    homepage,
```

Comment:
Sorry to clarify my comment here - I think these enum names should be capitalized, but they may need an additional lowercase string value like HOMEPAGE("homepage") if it's a requirement to pass the values lower cased. @jaxdesmarais do you know if they need to be case sensitive?

Reply:
https://paypal.atlassian.net/wiki/spaces/MI/pages/1038975282/Internal+Shopper+Insights+v1.5+-+BT+SDKs
<img width="833" alt="Screenshot 2024-12-18 at 10 55 50 AM" src="https://github.com/user-attachments/assets/9f7e5a9e-04eb-41fb-bfdd-c00938a44792" />